kong/plugins/grpc-gateway/handler.lua

```diff
@@ -24,11 +23,10 @@ local grpc_gateway = {
   VERSION = kong_meta.version,
 }
 
-
 local CORS_HEADERS = {
   ["Content-Type"] = "application/json",
   ["Access-Control-Allow-Origin"] = "*",
-  ["Access-Control-Allow-Methods"] = "GET,POST,PATCH,DELETE",
+  ["Access-Control-Allow-Methods"] = "GET,POST,PUT,PATCH,DELETE",
```

Comment:
This requires an entry in changelog (as a feature, we are now supporting PUT method, or as a bug fix, we corrected the header).

Reply:
May this be? https://github.com/Kong/kong/pull/8297/commits/20c2d67780ea36931edadb43aa195a6b1cc90a87